cubic-server/configuration/ConfigHandler.cpp

```diff
@@ -11,6 +13,20 @@ namespace Configuration
 
     void ConfigHandler::parse(const std::string &path) {
         YAML::Node config;
+        if (!std::filesystem::exists(path)) {
+            std::ofstream configFile(path);
+            constexpr std::string_view fileContent =
```

Comment:
Please put it outside of the function in the `Configuration` namespace, and call it something like `defaultConfigContent`